core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -882,7 +882,9 @@ $(document).ready(function () {
                                 if (typeof msg === 'undefined') {
                                     return "";
                                 } else {
-                                    var formHead = msg.substring(0, msg.indexOf("at"));
+                                    var indexOfLineSperator = msg.indexOf("\n");
+                                    var formHead = indexOfLineSperator > 0 ? msg.substring(0, indexOfLineSperator) : 
```

Comment:
nit: Could you remove the trailing white space?

Reply:
@sarutak to avoid the browser api compatibility issue, I  try to use a regular expressions instead of `String.prototype.trimRight` to handle  trailing while space.

Reply:
Ah, I meant `var formHead = indexOfLineSperator > 0 ? msg.substring(0, indexOfLineSperator) :[ ]` <- this trailing whitespace...
The latest change (trimming) is not necessary.

Reply:
Sorry for taking up so much of your time. I just woke up but my mind is still asleep... please help to review it thanks.